Source/AwsCommonRuntimeKit/mqtt/MqttRequestResponseClient.swift

```diff
@@ -79,11 +91,14 @@ public typealias SubscriptionStatusEventHandler = @Sendable (SubscriptionStatusE
 /// Function signature of an IncomingPublishEvent event handler
 public typealias IncomingPublishEventHandler = @Sendable (IncomingPublishEvent) -> Void
 
-/// Encapsulates a response to an AWS IoT Core MQTT-based service request
+/// A response to an AWS IoT Core MQTT-based service request
 public struct MqttRequestResponse: Sendable {
+  /// The MQTT Topic that the response was received on.
   public let topic: String
+  /// Payload of the response that correlates to a submitted request.
   public let payload: Data
 
+  /// Internal constructor that setup MqttRequestResponse from native aws_mqtt_rr_incoming_publish_event
```

Comment:
Trivial: Same as above. Or you can use "sets up" instead of "setup". It just shouldn't be "that setup <Thing>"